rosdep/python.yaml

```
@@ -6173,6 +6173,10 @@ python3-numpy:
   openembedded: [python3-numpy@openembedded-core]
   rhel: ['python%{python3_pkgversion}-numpy']
   ubuntu: [python3-numpy]
+python3-numpy-stl:
+  debian: [numpy-stl]
+  fedora: [python3-numpy-stl]
+  ubuntu: [python3-numpy-stl]
```

This package does not appear to be available on Ubuntu Xenial which is still a few months away from end-of-support. I could bend on this if another rosdep maintainer decides that It's Fine:tm:.
```suggestion
  ubuntu: 
    xenial: [null]
    '*': [python3-numpy-stl]
```